release-notes-1.2.1.md

```diff
@@ -0,0 +1,36 @@
+# Overview of jBallerina 1.2.1
+
+The jBallerina 1.2.1 patch release improves upon the 1.2.0 release by introducing the features listed below and addressing a number of [bugs](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A%22Ballerina+1.2.1%22+label%3AType%2FBug+is%3Aclosed) and [improvements](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A%22Ballerina+1.2.1%22+is%3Aclosed+label%3AType%2FImprovement).
+
+You can use the update tool to update to jBallerina 1.2.1 as follows.
+
+**For existing users:**
+
+If you are already using jBallerina version 1.2.0, you can directly update your distribution to jBallerina 1.2.1 by executing the following command:
+
+> $ ballerina dist update
+
+However, you need to use the following commands instead of the above if you have installed:
+
+- jBallerina 1.2.0 but switched to a previous version: `$ ballerina dist pull jballerina-1.2.1`
+- a jBallerina version below 1.1.0: install via the [installers](https://ballerina.io/downloads/)
+
+**For new users:**
+
+If you have not installed jBallerina, then download the [installers](https://ballerina.io/downloads/) to install.
+
+## Language
+- Deprecation support for annotations, function parameters and object fields
+- Runtime validations added to XML attributes map.
+- Support for map to record assignment
```

Comment:
```suggestion
- Support for a map to record the assignments
```

Reply:
This is actually correct. This refers to assigning map types to record types